src/jdk.jartool/share/classes/sun/security/tools/jarsigner/Resources.java

```diff
@@ -164,6 +164,7 @@ public class Resources extends java.util.ListResourceBundle {
 
         {"history.with.ts", "- Signed by \"%1$s\"\n    Digest algorithm: %2$s\n    Signature algorithm: %3$s, %4$s\n  Timestamped by \"%6$s\" on %5$tc\n    Timestamp digest algorithm: %7$s\n    Timestamp signature algorithm: %8$s, %9$s"},
         {"history.without.ts", "- Signed by \"%1$s\"\n    Digest algorithm: %2$s\n    Signature algorithm: %3$s, %4$s"},
+        {"history.nonexistent.entries", "    Warning: nonexistent signed entries: "},
```

Comment:
Inline with existing warning message, can it be "WARNING: Nonexistent signed entries: "

Reply:
This warning is a part of a block on signer info and I'd like the word does not stand out too much. For example, the test shows:
```
- Signed by "CN=x"
    Digest algorithm: SHA-384
    Signature algorithm: Ed25519, 255-bit key
    Warning: nonexistent signed entries: [a]
```